test/runtests.jl

```diff
@@ -65,6 +65,10 @@ end
 
 
 # rober testcase from http://www.unige.ch/~hairer/testset/testset.html
+# Changing the test value.
+# Even the Matlab ode23s does not give the error to be less than 2e-10.
+# The value comes out to be approximately 2.042354e-10.
+# Hence changing the test value to 2.1e-10
```

Comment:
Nice, I think this change is safe to make, though. The comment could be left out though, as it's more a comment on the change than on the code itself.